src/components/stripe-component/index.js

```diff
@@ -25,10 +25,11 @@ const StripeProvider = ({ userProfile, reservation, payTicket, providerKey, prov
     const stripePromise = useMemo(() => loadStripe(providerKey), [providerKey])
 
     const options = {
-        fonts: stripeOptions?.fonts,
+        // fonts: stripeOptions?.fonts,
         mode: 'payment',
-        currency: 'usd',
-        amount: 15
+        paymentMethodCreation: 'manual',        
```

Comment:
why this is needed ?

Reply:
This is required to create the payment method on the client with the data from the form and and get the token (now payment id) to confirm the payment 
https://github.com/fntechgit/summit-registration-lite/blob/702e0785694a6e0fc55c4d499ceb042ac00b3cce/src/components/stripe-form/index.js#L126